Dropdown/src/Dropdown.jsx

```diff
@@ -185,16 +185,17 @@ class Dropdown extends Component {
         const index = props.options.findIndex(this.findMatchingItem, this);
         if(index) {
 
-            const value = this.getOption(index).value;
+            const value = this.getOption(index);
```

Comment:
I think it should be called `option` now

Reply:
yes, you're right